docs/docs/labs/index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Overview"
+sidebar_class_name: hidden
+unlisted: true
+---
+
```

Comment:
non-blocking comment
1. would be nice to show the sub-sections as cards like https://docs.dagster.io/integrations/libraries/
2. it's a little weird when you go to the individual page like [this](https://dagster-docs-qj8uswlzj-elementl.vercel.app/labs/components/), this overview page becomes unlisted.
<img width="355" alt="image" src="https://github.com/user-attachments/assets/c24edbb9-2b1e-4eab-bcd5-b3999fbc3a58" />